lib/backpex/live_resource.ex

```diff
@@ -922,6 +925,10 @@ defmodule Backpex.LiveResource do
 
       def get_empty_filter_key, do: @empty_filter_key
 
+      def get_primary_key_field, do: @primary_key_field
+
+      defp primary_key(item), do: Map.get(item, @primary_key_field)
```

Comment:
We should define `primary_key(item)` and `primary_key(assigns, item)` next to each other:

```elixir
defp primary_key(item), do: Map.get(item, @primary_key_field)
defp primary_key(assigns, item), do: Map.get(item, assigns.primary_key_field)
```

Is the second function (with the assigns as the first param) even called? I cannot find any references to this function.

Reply:
aaaand done